packages/docs-page/components/version-alert/index.tsx

```diff
@@ -5,25 +5,24 @@ import {
   removeVersionFromPath,
 } from '../version-select/util'
 import s from './style.module.css'
```

Comment:
This cuts down some logic in this component, and makes it closer to a purely presentational component